Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -239,6 +239,10 @@ extension NSManagedObjectContext: MessageDatabaseSession {
             parentMessageDTO.replies.insert(message)
         }
         
+        channelDTO.lastMessageAt = createdDate
+        channelDTO.updatedAt = createdDate
+        channelDTO.defaultSortingAt = createdDate
```

Comment:
I just realized @nuno-vieira , this is not a wrong approach. When you send a message and you're offline the channel should still jump to the topmost position. So we actually really need something like this 👍 

Please add a comment there explainig why we do this + tests